src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -334,6 +418,10 @@ export default {
 			}
 		}, 250),
 
+		toggleNewGroupConversation(value) {
+			this.isNewGroupConversationOpen = value
+		},
+
```

Comment:
```suggestion
```

We don't have any components for this, as you removed button. It's fine, let's clear it here and implement in another PR